Solutions/Marain.Workflow.Api.Specs/Bindings/WorkflowFunctionsContainerBindings.cs

```diff
@@ -42,7 +42,9 @@ public static void SetupFeature(FeatureContext featureContext)
                     services.AddTenantProviderBlobStore();
 
                     services.AddTenantCosmosContainerFactory(root);
-                    services.AddWorkflowEngineFactory();
+                    services.AddTenantedWorkflowEngineFactory();
+                    services.AddTenantedAzureCosmosWorkflowStore(root);
```

Comment:
Is this how we name "CosmosDB" things elsewhere?

Reply:
We haven't been particularly consistent with other types of services - for example, in tenancy we have a project called `Marain.Claims.Tenancy.AzureBlob`, a service implementation called `BlobContainerPermissionsStoreFactory` (which is tenanted) and an `IServiceCollection` extension method called `TenantedBlobStorageClaimsServiceCollectionExtensions`

The naming I've used here aligns with that in Content Management, which is the only other Marain service we have that uses Cosmos at the moment. Arguably we could use `CosmosDb` instead of just `Cosmos` but I don't think it adds anything so I'm happy leaving it as is in both repos.